unit-tests/post-processing/test-post-processing-from-bag.py

```diff
@@ -0,0 +1,126 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#temporary fix to prevent the test from running on Win_SH_Py_DDS_CI 
+#test:donotrun:dds
+
+import pyrealsense2 as rs
+from rspy import test, repo
+import os.path
+import time
+################################################################################################
+pre_aligned_frames_map ={}
+frames_data_map = {}
+playback_status = None
+sensors = []
+align_from = rs.stream(rs.stream.depth)
+align = rs.align(rs.stream.color)
+
+
+def playback_callback(status):
+    global playback_status
+    playback_status = status
+
+
+def validate_ppf_results(result_frame_data, reference_frame_data):
+    result_bytearray_frame_data, result_profile = result_frame_data
+    reference_bytearray_frame_data, reference_profile = reference_frame_data
+
+    test.check_equal(result_profile.width(), reference_profile.width())
+    test.check_equal(result_profile.height(), reference_profile.height())
+    test.check_equal_lists(result_bytearray_frame_data,reference_bytearray_frame_data)
+
+
+def process_frame(frame, frame_source):
```

Comment:
Right now it's `align`-specific. What do we have to do to make it generic, so next time we do this test it can work with `pointcloud`?

Reply:
Done